src/AWS.Deploy.Orchestration/Utilities/DeployedApplicationQueryer.cs

```diff
@@ -73,7 +87,32 @@ public async Task<List<CloudApplication>> GetExistingDeployedApplications(IList<
                     continue;
                 }
 
-                apps.Add(new CloudApplication(stack.StackName, recipeId));
+                apps.Add(new CloudApplication(stack.StackName, recipeId, stack.LastUpdatedTime));
```

Comment:
What time zone is `stack.LastUpdatedTime` returned in? Also, as per the documentation, this field will only be returned if the stack has been updated at least once. 

![image](https://user-images.githubusercontent.com/36622308/129274453-6e2e176e-4f04-443d-9c9f-e03394d6c97e.png)

source - https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/CloudFormation/TStack.html

Will this be a problem?

Reply:
I checked and every stack deployed by the deploy tool has the Last Updated Time. I also tried to fetch them with the API and it was fine.